src/herder/test/HerderTests.cpp

```diff
@@ -3720,14 +3679,6 @@ herderExternalizesValuesWithProtocol(uint32_t version)
             Herder::State::HERDER_BOOTING_STATE);
 
     simulation->startAllNodes();
-    if (protocolVersionStartsFrom(version, SOROBAN_PROTOCOL_VERSION))
```

Comment:
The loadgen upgrade helper function is a little hacky and assumes it will only be called once. While it's probably a good idea to fix eventually, this PR was big enough as it was. Everything before this point is set up anyway, so there was no reason (as far as I could tell) why we couldn't combine the upgrades into a single upgrade to play better with our test helpers. It doesn't need to be protocol gated anymore since this is herder tests and we'll never externalize a pre p20 ledger again.

Reply:
Hmm it was actually pretty quick to fix the upgrade function so I'll return back to the original test with two different upgrades.